profiles/factory.py

```diff
@@ -47,6 +47,6 @@ def create_app(config: Config, clients: Clients) -> Flask:
     app.register_error_handler(ClientError, errors.client_error_handler)
     app.register_error_handler(OAuth2Error, errors.oauth2_error_handler)
 
-    models_committed.connect(send_update_events(publisher=publisher), app, weak=False)
+    models_committed.connect(send_update_events(publisher=publisher), weak=False)
```

Comment:
Is it possible to test this?

Reply:
If we had a route to POST to whose only behavior was to do an empty update of a `Profile` (bumping its updated date, if there is one?), we could do a smoke test or a manual test that would work on `dev` and `ci`.

Otherwise this will be tested in end2end, but we have no checks in automated test yet for that as there is no observable behavior depending on the event (that I have seen).

Reply:
I could take another look, but this was a sticking point last time. Events just don't seem to get fired in the same way when using test fixtures, when run using a dummy app (live not test suite) it works. Something to do with the setup/teardown + blinker. That's why we wanted some kind of end2end test(s) for this.

Reply:
I take that back. Replacing Signal.ANY was, in fact, the solution for everything. Can easily test this now. Will add additional tests.

Thanks @thewilkybarkid